subprojects/dependency-management/src/test/groovy/org/gradle/api/internal/catalog/DefaultVersionCatalogBuilderTest.groovy

```diff
@@ -33,7 +36,24 @@ import java.util.function.Supplier
 class DefaultVersionCatalogBuilderTest extends Specification implements VersionCatalogErrorMessages {
 
     @Subject
-    DefaultVersionCatalogBuilder builder = new DefaultVersionCatalogBuilder("libs", Interners.newStrongInterner(), Interners.newStrongInterner(), TestUtil.objectFactory(), Stub(Supplier))
+    DefaultVersionCatalogBuilder builder = createVersionCatalogBuilder()
+
+    //reuse!!!
```

Comment:
:x: Was it reused?
Nevertheless, this comment should be either resolved or deleted